mesheryctl/cmd/vars.go

```diff
@@ -92,6 +92,10 @@ func setFileLocation() {
 
 //Pre-Flight Check
 func preReqCheck() {
+	//No checks for Docker-compose for windows
+	if runtime.GOOS == "windows" {
+		return
+		}
 	//Check for installed docker-compose on client system
```

Comment:
@anandrkskd you'll want to double-check on a Windows machine, but I think we will still want to run a check for `docker-compose` as a prerequisite, let the user know if a prerequisite is unmet, but just not try to automatically install that unmet prereq (not run `installprereq()`).

Reply:
Understood